### PR TITLE
system-variables: add doc for tidb_expensive_txn_time_threshold

### PR DESCRIPTION
**Pull request by crazycs520**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/13848
- Other reference link(s): https://github.com/pingcap/tidb/pull/44248

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -2104,6 +2104,16 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
 
+### `tidb_expensive_txn_time_threshold`
+
+- 作用域：GLOBAL
+- 是否持久化到集群：否，仅作用于当前连接的 TiDB 实例
+- 类型：整数型
+- 默认值：`600`
+- 范围：`[60, 2147483647]`
+- 单位：秒
+- 这个变量用来控制打印 expensive txn 日志的阈值时间，默认值是 600 秒。expensive txn 日志会把正在执行中的事务且执行时间超过阈值的事务相关信息打印出来。
```
Comment (cfzjywxk) [inline system-variables.md:2115]: 需要说明 `expensive txn` 的具体定义

Comment (crazycs520) [reply system-variables.md]: done.

**Review by cfzjywxk — APPROVED** [review]
LGTM

File: system-variables.md [hunk 2]
```diff
@@ -2104,6 +2104,16 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 单位：秒
 - 这个变量用来控制打印 expensive query 日志的阈值时间，默认值是 60 秒。expensive query 日志和慢日志的差别是，慢日志是在语句执行完后才打印，expensive query 日志可以把正在执行中的语句且执行时间超过阈值的语句及其相关信息打印出来。
 
+### `tidb_expensive_txn_time_threshold`
```
Comment (Oreoxmt) [inline system-variables.md:2107]:
```suggestion
### `tidb_expensive_txn_time_threshold` <span class="version-mark">从 v7.2.0 版本开始引入</span>
```

File: system-variables.md [hunk 3]
```diff
@@ -2104,6 +2104,16 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
 
+### `tidb_expensive_txn_time_threshold` <span class="version-mark">从 v7.2.0 版本开始引入</span>
+
+- 作用域：GLOBAL
+- 是否持久化到集群：否，仅作用于当前连接的 TiDB 实例
+- 类型：整数型
+- 默认值：`600`
+- 范围：`[60, 2147483647]`
+- 单位：秒
+- 这个变量用来控制打印 expensive txn 日志的阈值时间，默认值是 600 秒。expensive txn 日志会把未 COMMIT/ROLLBACK 的事务且持续时间超过阈值的事务相关信息打印出来。
```
Comment (Oreoxmt) [inline system-variables.md:2115]:
```suggestion
- 这个变量用来控制打印 expensive transaction 日志的阈值时间，默认值是 600 秒。expensive transaction 日志会把未 COMMIT/ROLLBACK 的事务且持续时间超过该阈值的事务相关信息打印出来。
```

File: system-variables.md [hunk 4]
```diff
@@ -2104,6 +2104,16 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
 
+### `tidb_expensive_txn_time_threshold` <span class="version-mark">从 v7.2.0 版本开始引入</span>
+
+- 作用域：GLOBAL
+- 是否持久化到集群：否，仅作用于当前连接的 TiDB 实例
+- 类型：整数型
+- 默认值：`600`
+- 范围：`[60, 2147483647]`
+- 单位：秒
+- 这个变量用来控制打印 expensive transaction 日志的阈值时间，默认值是 600 秒。expensive transaction 日志会把未 COMMIT/ROLLBACK 的事务且持续时间超过该阈值的事务相关信息打印出来。
```
Comment (qiancai) [inline system-variables.md:2115]:
```suggestion
- 这个变量用来控制打印 expensive transaction 日志的阈值时间，默认值是 600 秒。expensive transaction 日志会将尚未 COMMIT 或 ROLLBACK 且持续时间超过该阈值的事务的相关信息打印出来。
```

**Review by qiancai — APPROVED** [review]
/lgtm

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**